### PR TITLE
Merge asm patch

### PR DESCRIPTION
**Pull request by elfinhe**
Merge asm-patch into release 1.5 branch.

### CODE REVIEW

File: asm-patch/Kptfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+packageMetadata:
+  shortDescription: sample description
+openAPI:
+  definitions:
+    io.k8s.cli.setters.gcloud.core.project:
+      x-k8s-cli:
+        setter:
+          name: gcloud.core.project
+          value: PROJECT_ID
+    io.k8s.cli.setters.gcloud.container.cluster:
```
Comment (nan-yu) [inline asm-patch/Kptfile:14]: Thanks for unifying the cluster setter name!

**Comment by elfinhe** [comment]
> LGTM. Do you want to wait after #48 is merged and rebase this PR to include the update validation function in release-1.5?

Great, I will wait for that. Thanks Nan for the quick review.

**Comment by nan-yu** [comment]
@elfinhe #48 has been merged. You can rebase this PR now. Thanks!

**Comment by elfinhe** [comment]
Replaced by https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/50

**Pull request closed without merging**